lib/recurly/base.php

```diff
@@ -112,6 +112,27 @@ protected static function _delete($uri, $client = null)
     return null;
   }
 
+  /**
+   * Returns URI for resource, throwing error if resource code is an empty string
+   * @param string First Recurly Client path
+   * @param string Resource codes and paths
+   * @return string URI
+   * @throws Recurly_Error
+   */
+  public static function _safeUri($req, ...$params) {
+    $path = "";
```

Comment:
Maybe just initialize this as `$path = $req`. Then on the bottom you `return $path`

Reply:
Comment not relevant anymore.